hm_pyhelper/miner_param.py

```diff
@@ -59,9 +59,14 @@ def provision_key():
     direct_path = os.path.dirname(os.path.abspath(__file__))
     gateway_mfr_path = os.path.join(direct_path, 'gateway_mfr')
 
-    if get_gateway_mfr_test_result():
-        logging.info("Key already provisioned")
-        return True
+    test_results = get_gateway_mfr_test_result()
```

Comment:
I'm going to write a test case and push a fix because I think this is still a little off for two reasons.

`get_gateway_mfr_test_result` returns something like this:

```
{
  "result": "pass",
  "tests": [
    {
      "output": "ok",
      "result": "pass",
      "test": "serial"
    },
    ...
```

The top level result will only `pass` [if all of the individual tests also pass](https://github.com/helium/gateway-mfr-rs/blob/main/src/cmd/test.rs#L36). That means its redudant to check if `miner_key(0)` passed after already checking the top level.